RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -71,14 +72,13 @@
     CAThetaCut = 0.0012,
     CAPhiCut = 0.2,
 )
+
 highBetaStar_2018.toModify(_initialStepCAHitQuadruplets,
     CAThetaCut = 0.0024,
     CAPhiCut = 0.4
 )
 initialStepHitQuadruplets = _initialStepCAHitQuadruplets.clone()
 
-trackingPhase1.toModify(initialStepHitDoublets, layerPairs = [0,1,2]) # layer pairs (0,1), (1,2), (2,3)
-
```

Comment:
Could you undo this change, or explain why it is needed? (to me the new placement is not really better than this one)